test/compilable/previewall.d

```diff
@@ -0,0 +1,8 @@
+// PERMUTE_ARGS: -preview=all -transition=all -revert=all
```

Comment:
Does this need to run permutations? Wouldn't `ARG_SETS` suffice?

Reply:
Thanks, moved to `ARG_SETS`